Extensions/TileMap/JsExtension.js

```diff
@@ -903,7 +918,7 @@ const defineSimpleTileMap = function (extension, _, gd) {
       _('Flip tile vertically (on the grid)'),
       _('Flip tile vertically at grid coordinates.'),
       _(
-        'Flip tile vertically at grid coordinates _PARAM1_ ; _PARAM2_: _PARAM3_'
+        'Flip tile vertically at grid coordinates _PARAM1_ ; _PARAM2_: _PARAM3_ of _PARAM0_'
```

Comment:
```suggestion
        'Flip tile vertically at grid coordinates _PARAM1_;_PARAM2_ of _PARAM0_: _PARAM3_'
```

Reply:
I prefer spaces around the semicolon. Otherwise, expressions are hard to read.
![image](https://github.com/user-attachments/assets/400927a1-8a49-4e0b-a1a4-efd86b5831c5)